README.md

```diff
@@ -186,6 +186,54 @@ To run the Sass linter (stylelint) before commits, run the following command: fr
 
 Full details of stylelint are available here: https://stylelint.io/
 
+### Running unit tests
+
+We have two types of unit tests: Spec tests for our Scala code, which are located within the `test > controllers` and
```

Comment:
I would prefer to add `test` only. ie. which are located within the `test` directories .....

because there are many sub directories in the test.